rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.desired_balance/10_basic.yml

```diff
@@ -73,38 +73,38 @@ setup:
   - do:
       _internal.get_desired_balance: { }
 
-  - is_true: 'cluster_balance_stats'
-  - is_true: 'cluster_balance_stats.tiers'
-  - is_true: 'cluster_balance_stats.tiers.data_content.shard_count'
-  - is_true: 'cluster_balance_stats.tiers.data_content.shard_count.total'
-  - is_true: 'cluster_balance_stats.tiers.data_content.shard_count.min'
-  - is_true: 'cluster_balance_stats.tiers.data_content.shard_count.max'
-  - is_true: 'cluster_balance_stats.tiers.data_content.shard_count.average'
-  - is_true: 'cluster_balance_stats.tiers.data_content.shard_count.std_dev'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_write_load'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_write_load.total'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_write_load.min'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_write_load.max'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_write_load.average'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_write_load.std_dev'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_disk_usage'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_disk_usage.total'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_disk_usage.min'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_disk_usage.max'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_disk_usage.average'
-  - is_true: 'cluster_balance_stats.tiers.data_content.forecast_disk_usage.std_dev'
-  - is_true: 'cluster_balance_stats.tiers.data_content.actual_disk_usage'
-  - is_true: 'cluster_balance_stats.tiers.data_content.actual_disk_usage.total'
-  - is_true: 'cluster_balance_stats.tiers.data_content.actual_disk_usage.min'
-  - is_true: 'cluster_balance_stats.tiers.data_content.actual_disk_usage.max'
-  - is_true: 'cluster_balance_stats.tiers.data_content.actual_disk_usage.average'
-  - is_true: 'cluster_balance_stats.tiers.data_content.actual_disk_usage.std_dev'
-  - is_true: 'cluster_balance_stats.nodes'
-  - is_true: 'cluster_balance_stats.nodes.$node_name'
-  - gte: { 'cluster_balance_stats.nodes.$node_name.shard_count' : 0 }
-  - gte: { 'cluster_balance_stats.nodes.$node_name.forecast_write_load': 0.0 }
-  - gte: { 'cluster_balance_stats.nodes.$node_name.forecast_disk_usage_bytes' : 0 }
-  - gte: { 'cluster_balance_stats.nodes.$node_name.actual_disk_usage_bytes' : 0 }
```

Comment:
I think we want to leave these as `gte` assertions - that way we're saying that these things are present _and_ numeric _and_ nonnegative.